packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -595,97 +590,91 @@ describe('src/cy/commands/querying', () => {
         it('returns an array of xhrs', () => {
           cy
           .visit('http://localhost:3500/fixtures/jquery.html')
-          .server()
-          .route(/users/, {}).as('getUsers')
+          .intercept(/users/, {}).as('getUsers')
           .window().then({ timeout: 2000 }, (win) => {
             return Promise.all([
               win.$.get('/users', { num: 1 }),
               win.$.get('/users', { num: 2 }),
             ])
-          }).get('@getUsers.all').then((xhrs) => {
+          }).wait('@getUsers').wait('@getUsers').get('@getUsers.all').then((xhrs) => {
```

Comment:
is there a better way to wait twice?

Reply:
Not until detached dom is merged. Once that's ready, you'll be able to do something like I suggested above:
```
cy.get('@getUsers.all').should(xhrs => {
  expect(xhrs).to.have.length(2)
  ...etc...
})
```

Switching from .then() to .should() means it'll retry incoming queries (eg, by refetching `getUsers.all`).

Reply:
should we add a TODO to update this test once the detached dom work merges in?